pages/dkp/kommander/2.1/backup-and-restore/index.md

```diff
@@ -73,6 +76,31 @@ data:
         #   aws_secret_access_key=<REDACTED>
 ```
 
+After you have configured the `ConfigMap` with **MinIO**, OR **Amazon S3**, add the `configOverrides` value to the Velero `AppDeployment`. This applies the `ConfigMap` to your instance after the cluster is configured:
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: apps.kommander.d2iq.io/v1alpha2
+kind: AppDeployment
+metadata:
+  creationTimestamp: "2022-06-10T04:46:31Z"
+  finalizers:
+  - kommander.mesosphere.io/appdeployment
+  - kommander.mesosphere.io/appsconfigfederation
+  generation: 1
+  name: velero
+  namespace: kommander
+  resourceVersion: "27126"
+  uid: 76f4852b-8207-43fb-8a8c-5e7b4802696c
+spec:
+  appRef:
+    kind: ClusterApp
+    name: velero-3.1.5
+  configOverrides:
+    name: velero-overrides
+EOF
+```
```

Comment:
Some of the values here are specific to a cluster where this `AppDeployment` was created. I think that universal way to update the `AppDeployment` would be to use `kubectl patch` instead of `kubectl apply`

```
cat << EOF | kubectl -n kommander patch appdeployment velero --type="merge" --patch-file=/dev/stdin
spec:
  configOverrides:
    name: velero-overrides
EOF
```